src/main/resources/messages.properties

```diff
@@ -61,7 +61,7 @@ date = Approval date:
 
 # Validation
 validation.range.outside = Enter a value between {0} and {1}
-validation.character.invalid = Enter a value without commas, brackets or other symbols
+validation.characters.invalid = Enter a value without commas, brackets or other symbols
```

Comment:
I'm not sure this is needed anymore. The 'false' in the enum above indicates that this error is not generic. As a result, the message resolver will look for a message of the format `validation.characters.invalid.<json_path_of_field_in_error>`.